src/customJs/tools/product/index.ts

```diff
@@ -33,6 +33,12 @@ const productLayoutProperty: () => UnlayerProperty<ProductLayout> = () =>
     ],
   } as const);
 
+const isEmptyOrZero = (value: string) => {
+  if (value === '') return true;
+  const numberValue = value.split(' ')[1] || '0';
```

Comment:
Hi @ck-makingsense!

I am glad you are working with the Editor WebApp again, and it seems that you are doing great!

I am not sure about this line, it seems a little confusing to me, see some examples:

```
"$  1".split(' ')[1] // ""
"$1".split(' ')[1]   // undefined
"1".split(' ')[1]    // undefined
"$1".split(' ')[1]   // undefined
"0 €".split(' ')[1]  // €
```

Maybe you can add unit tests for `isEmptyOrZero` function.

Reply:
Only to contextualize a bit, the prices now are formatted in [the backend by Doppler](https://github.com/MakingSense/Doppler/blob/develop/Doppler.Application.ControlPanelModule/Services/Interfaces/EcommerceAbstractService.cs#L55) (similar to the [format used in the MSEditor](https://github.com/MakingSense/MSEditor/blob/master/app/partials/directives/mseditorProductCard.html#L11)).

The possible currency symbols are [the following](https://github.com/MakingSense/Doppler/blob/develop/Doppler.Application.ControlPanelModule/Services/Interfaces/EcommerceAbstractService.cs#L8): `$, U$D, €, R$, S/`
([according to the MSEditor](https://github.com/MakingSense/MSEditor/blob/master/app/constants.js#L79), too).

Maybe would be better that we use a regular expression to validate empty or zero?

Reply:
Thank you @leoslopez. 

I understand that now the expected values are being sanitized by Doppler Backend, but based on the [Robustness Principle](https://en.wikipedia.org/wiki/Robustness_principle) (_"be conservative in what you send, be liberal in what you accept"_) I think that it should be ready to accept other formats, for example, values without space of using the symbol as a suffix.

Reply:
Andres tal cual, esto depende demasiado del formato que me traiga el backend. Por ahi me parece que conviene cambiar la estratagia

Reply:
O simplemente dejarla explícita en un test.